home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -498,7 +498,118 @@ paths:
           $ref: '#/components/responses/NotFound'
         default:
           $ref: '#/components/responses/InternalServerError'
-
+  /cooperation/{id}/tasks:
+    post:
```

Comment:
In Issue#292 for these method and endpoint another requestBody is provided. It is need to decide which of these options is correct.